src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int STACK_CAPACITY = 12;
+    private final int[] NUMBERS = new int[STACK_CAPACITY];
+    private static final int RETURN_VALUE = -1;
+
+    private static final int FULL_TOTAL = 11;
+
+    private static final int CHECK_TOTAL = -1;
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = -1;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).